lib/index.js

```diff
@@ -10,6 +10,8 @@ const pathExists = require('path-exists')
 const semver = require('semver')
 const sumchecker = require('sumchecker')
 
+let tmpFileCounter = 0
```

Comment:
> a class variable, not an instance variable.

Hmm... ES6 doesn't support 

```
static tmpFileCounter = 0
```

> SyntaxError: Unexpected token =

We cannot use this syntax because node doesn't support it.

What do you mean? And for what we should use class variable here? It should be module level variable. e.g. I write in Kotlin and prefer to use file level vars instead of static.